app/controllers/admin/loan_questions_controller.rb

```diff
@@ -72,9 +72,12 @@ def set_loan_question
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def loan_question_params
-      params.require(:custom_field).permit(:label, :data_type, :parent_id, :position,
+      params.require(:custom_field).delete_if { |k, v| k =~ /^locale_/ }.permit(
```

Comment:
@linesarefuzzy I'm assuming you put the `delete_if` here? Can you please explain?